### PR TITLE
docs(2021.2): Update release note for 7.13.7

### PR DESCRIPTION
**Pull request by VishalThapaliya**
Add bug fixes

### CODE REVIEW

**Comment by vbotnari12** [comment]
why did you decided to add "Fix" in front of the bugs? :) I dont mind :)

**Comment by VishalThapaliya** [comment]
> why did you decided to add "Fix" in front of the bugs? :) I dont mind :)

We used the term "Fix" because we saw that the dev suite team was doing it and we found this is a good practice for the bug fixes to differentiate them from improvements and dependency updates.

**Review by abirembaut — APPROVED** [review]
LGTM

File: modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -234,19 +234,38 @@ If you update Bonita from an earlier version, you can now delete them if you do
...
+* STUDIO-4437 - Fix SNAPSHOT external lib not displayed in "add dependency" in process configuration
+
+==== Fixes in Bonita Runtime including Bonita Applications
+
+* RUNTIME-145 - Fix Create User on the fly when Role doesn't exist
+* RUNTIME-752 - Fix Error 404 resulting in blank task list
+* RUNTIME-1431 - Fix Instantiation REST service Exposing the groovy script behind an exception
+* RUNTIME-1434 - Fix Custom user information truncated in admin app
+* RUNTIME-1449 - [Security] Update several dependencies for 2021.2-u7 (7.13.7)
+* RUNTIME-1476 - Fix Default dynamic permissions: process deployers, process initiators and process managers can access GET BPM/process
```
Comment (abirembaut) [inline modules/ROOT/pages/release-notes.adoc]: should remain bpm as it is a URL
```suggestion
* RUNTIME-1476 - Fix Default dynamic permissions: process deployers, process initiators and process managers should be able to access GET bpm/process
```

**Comment by vbotnari12** [comment]
I am taking into account the content check from vale. It's a new check, in beta test, and an important one :) it's our grammar police

**Pull request merged**